pages/members/Member.js

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import styles from "./members.module.css"
+import Image from "next/image"
+import bannerImgDefault from "../../public/blank_banner.png"
+
+export default function Member(props) {
```

Comment:
You can deconstruct `props` into `{profile_image_url, description}` so you don't have to do `props.XYZ` everytime.

Reply:
I'm pulling in 5 different props here. Is it worth to deconstruct always no matter what amount there are?